pkg/sidecar/docker_reactor.go

```diff
@@ -18,6 +18,16 @@ import (
 	"github.com/ipfs/testground/sdk/runtime"
 )
 
+// PublicAddr points to an IP address in the public range. It helps us discover
+// the IP address of the gateway (i.e. the Docker host) on the control network
+// (the learned route will be via the control network because, at this point,
+// the only network that's attached to the container is the control network).
+//
+// Sidecar doesn't whitelist traffic to public addresses, but it special-cases
+// traffic between the container and the host, so that pprof, metrics and other
+// ports can be exposed to the Docker host.
+var PublicAddr = net.ParseIP("1.1.1.1")
```

Comment:
This is the IP address of Cloudfliar DNS. I don't have a problem with that, just pointing that out.

Reply:
Yep, that's ok. This will not perform any lookup of any kind, it just matches against the kernel routing tables.